experimental/segmenter/tests/testdata/Thai_graphclust_exclusive_model4_heavy/converter.py

```diff
@@ -0,0 +1,12 @@
+# A Converter of weights.json to use ICU4X's data since the keys have to be sorted.
```

Comment:
Nit: maybe add `#!/usr/bin/env python3` at the first line, and `chmod +x converter.py` so that it's executable right away.